### PR TITLE
chore: pin copyright year in generated code

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7573)

### CODE REVIEW

**Review by dbolduc — APPROVED** [review]
Thanks

File: google/cloud/bigtable/admin/integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ set(bigtable_admin_integration_tests
 
 include(CreateBazelConfig)
 export_list_to_bazel("bigtable_admin_integration_tests.bzl"
-                     "bigtable_admin_integration_tests")
+                     "bigtable_admin_integration_tests" YEAR 2021)
```
Comment (devbww) [inline google/cloud/bigtable/admin/integration_tests/CMakeLists.txt:26]: Nit: My version of this also tweaked other instances, where about 75% of the time we double-quote the year.

Comment (coryan) [reply google/cloud/bigtable/admin/integration_tests/CMakeLists.txt]: Good point, I think this is now fixed across all files. PTAL

**Review by devbww — APPROVED** [review]
Thanks.

**Pull request merged**